api/.project/devstart.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "common/common"
```

Comment:
It may be useful to name it something related to what the scrips inside are planned to do. (docker-images or something?)